proof/refine/ARM/Syscall_R.thy

```diff
@@ -351,50 +351,49 @@ lemma threadSet_tcbDomain_update_sch_act_wf[wp]:
    apply (auto simp: obj_at'_def)
   done
 
-lemma set_domain_setDomain_corres:
+lemma valid_tcb_domain_update:
+  "valid_tcb tptr (tcb\<lparr>tcb_domain := new_dom\<rparr>) s = valid_tcb tptr tcb s"
```

Comment:
I'm not sure where to, but I think that these two lemmas should either be moved earlier or at least have a `FIXME: move` on them.